mysqldsb/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This pom should be the projects root pom..

Reply:
Thanks for fixing